src/components/App/App.js

```diff
@@ -9,41 +8,74 @@ function App() {
 
   const [movies, setMovies] = useState(movieData);
 	const [showMovieInfo, setShowMovieInfo] = useState(false);
+  const [movieId, setMovieId] = useState(0);
+	const [singleMovieInfo, setSingleMovieInfo] = useState({});
+  const [error, setError] = useState("")
+  
 
-	const handleMovieClick = () => {
+  function handleMovieClick() {
 		setShowMovieInfo(true);
-		console.log("hit handleMovieClick");
+  }
+
+	function handleError(error) {
```

Comment:
are these three wrapper functions needed? Maybe for passing props? They all look super similar, and they dont really do anything besides calling another function.